icedata/datasets/pets/parser.py

```diff
@@ -36,7 +36,7 @@ def labels(self, o) -> List[Hashable]:
         # there is an image with two cats (same breed)
         num_objs = len(self._root.findall("object"))
 
-        return [class_id] * num_objs
```

Comment:
the line that gets `class_id` can also be removed

Reply:
good point!
Done!